### PR TITLE
Incremental/x 4 create service

### PR DESCRIPTION
**Pull request by algpuentes**

### CODE REVIEW

File: src/main/java/com/xantar/xantarcore/db/ScheduleService.java [hunk 1]
```diff
@@ -1,9 +1,57 @@
...
+		.forEach((slotId, configurations) -> {
+			eMealsMap.put(slotId, this.mealsRepository.findMealsBySlotAndMaxRows(slotId, configurations.size()));
+		});
+
+
+		daysList.stream()
+		.forEach( day -> day.configurations
+				.forEach(config -> {
+					if(eMealsMap.get(config.retrieveSlotId()).size() > 0) {
+						final int index = (int) (Math.random() * (eMealsMap.get(config.retrieveSlotId()).size() - 0));
```
Comment (fabioir) [inline src/main/java/com/xantar/xantarcore/db/ScheduleService.java:48]: This introduces redundancy in the meals result. Can we better iterate over the DB returned results? If the number of results is not high enough then we may need to introduce redundancy, in every other case I see no point.

File: src/main/java/com/xantar/xantarcore/schedule/ScheduleController.java [hunk 2]
```diff
@@ -34,10 +34,11 @@
 
 
 	@PostMapping()
-	public ResponseEntity<List<ScheduleDayJson>> createMeal(@RequestBody List<ScheduleDayJson> jMeal) {
-		final var scheduleList = new ArrayList<ScheduleDayJson>();
-
-		return new ResponseEntity<List<ScheduleDayJson>>(scheduleList, HttpStatus.CREATED);
+	public ResponseEntity<ScheduleJson> createMeal(@RequestBody List<ScheduleDayJson> jDayList) {
```
Comment (fabioir) [inline src/main/java/com/xantar/xantarcore/schedule/ScheduleController.java]:
```suggestion
	public ResponseEntity<ScheduleJson> createSchedule(@RequestBody List<ScheduleDayJson> jDayList) {
```

**Review by fabioir — APPROVED** [review]
It looks good to me for an MVP, I would like to discuss further about the schedule generation and the testing coverage. Good Job! 🎉

File: src/main/java/com/xantar/xantarcore/db/ScheduleService.java (hunk 1 above)
Comment (algpuentes) [reply src/main/java/com/xantar/xantarcore/db/ScheduleService.java:48]: To take in consideration for the next iteration

**Pull request merged**